code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -256,9 +259,23 @@
 
 /mob/living/simple_animal/pet/cat/cak/attack_hand(mob/living/L)
 	..()
-	if(L.a_intent == INTENT_HARM && L.reagents && !stat)
-		L.reagents.add_reagent("nutriment", 0.4)
-		L.reagents.add_reagent("vitamin", 0.4)
+	if(L.a_intent != INTENT_HARM && L.a_intent != INTENT_DISARM)
+		return
```

Comment:
Why not have grab be able to eat aswell?

Reply:
Because this matches the behavior of the actual attack: https://github.com/ParadiseSS13/Paradise/blob/e1614b9ecbd10d92f7f461b81eec500c4c9c6e55/code/modules/mob/living/simple_animal/animal_defense.dm#L20